core/arch/arm/kernel/entry_a64.S

```diff
@@ -56,11 +56,20 @@
 	.endm
 
 FUNC _start , :
+#if defined(CFG_CORE_SEL1_SPMC)
+	/*
+	 * With OP-TEE as SPMC at S-EL1 the SPMD (SPD_spmd) in TF-A passes
+	 * the DTB in x0 and the rest of the registers are unused
+	 */
+	mov	x19, xzr	/* Pagable part not set */
```

Comment:
Does it mean CFG_CORE_SEL1_SPMC cannot be used with pager enabled?

Reply:
You're right. I've changed my mind, I'll enable pager support too.